<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceRecognition</class>
 <widget class="QMainWindow" name="FaceRecognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>759</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>759</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FaceRecognition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ViewLabel" name="previewLabel">
      <property name="minimumSize">
       <size>
        <width>436</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>436</width>
        <height>300</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="registerListWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="gridSize">
       <size>
        <width>70</width>
        <height>100</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTextEdit" name="editOutLog">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>141</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>741</width>
        <height>141</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>81</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>741</width>
        <height>81</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="btnClear">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>30</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Clear Faces</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRegister">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>30</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Register Face</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSelectImage">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Select Image</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCompareFace">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Compare Face</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOperateCamera">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Open Camera</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>LiveLine:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Threshold">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>10</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>FaceLine:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="faceThreshold">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>40</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0.7</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewLabel</class>
   <extends>QLabel</extends>
   <header location="global">viewlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
